### PR TITLE
Button

### PR DESCRIPTION
**Pull request by abdul-moeed-khan**
created a button file and added first button in it. Please review

### CODE REVIEW

File: src/Button.jsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+function Button() {
+  return (
+    <div>
+      <button>button1</button>
```
Comment (huzaifanaeem909) [inline src/Button.jsx]: Perferct

Comment (abdul-moeed-khan) [reply src/Button.jsx]: ok thanks

Comment (huzaifanaeem909) [reply src/Button.jsx]: Add some styling in button tag.

Comment (abdul-moeed-khan) [reply src/Button.jsx]: ok i will add it in a day

File: src/Button.jsx [hunk 2]
```diff
@@ -3,7 +3,7 @@ import React from 'react'
 function Button() {
   return (
     <div>
-      <button>button1</button>
```
Comment (abdul-moeed-khan) [inline src/Button.jsx]: Done

File: src/Button.jsx [hunk 3]
```diff
@@ -3,7 +3,7 @@ import React from 'react'
...
+      <button style="background:blue">button1</button>
```
Comment (huzaifanaeem909) [inline src/Button.jsx:6]: Change color from blue to yellow.

Comment (huzaifanaeem909) [reply src/Button.jsx:6]: Hello

Comment (abdul-moeed-khan) [reply src/Button.jsx:6]: ok

**Pull request merged**